{"name": "test_loginFailMethod", "status": "failed", "statusDetails": {"message": "AssertionError", "trace": "self = <AppiumFramwork.tests.LoginTest.LoginTest testMethod=test_loginFailMethod>\n\n    @pytest.mark.run(order=1)\n    def test_loginFailMethod(self):\n        cl.allureLogs(\"App Opened\")\n        self.gt.clickLoginBotton()\n        self.gt.enterEmail()\n        self.gt.enterPassword2()\n        self.gt.clickOnLoginB()\n>       self.gt.verifyAdminScreen()\n\nLoginTest.py:37: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <AppiumFramwork.pages.LoginPage.LoginPageTest object at 0x000001E57E75B6D0>\n\n    def verifyAdminScreen(self):\n        adminScreen = self.isDisplayed(self._pageTitle,\"text\")\n>       assert adminScreen == True\nE       AssertionError\n\n..\\pages\\LoginPage.py:44: AssertionError"}, "steps": [{"name": "App Opened", "status": "passed", "start": 1653572140643, "stop": 1653572140643}, {"name": "Click on Login Button", "status": "passed", "start": 1653572141409, "stop": 1653572141409}, {"name": "Entered email id", "status": "passed", "start": 1653572142995, "stop": 1653572142995}, {"name": "Entered Password", "status": "passed", "start": 1653572144112, "stop": 1653572144113}, {"name": "Clicked on Login Button in Login Screen", "status": "passed", "start": 1653572144693, "stop": 1653572144693}], "attachments": [{"name": "text", "source": "e73f3006-488e-4685-9e3e-288b38d1ccb1-attachment.png", "type": "image/png"}, {"name": "text", "source": "ad67dc79-6562-498e-8ce6-609a0f63ed99-attachment.png", "type": "image/png"}, {"name": "log", "source": "ef1adf14-f7d6-469e-9af3-e1b6116c0323-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "bbe0fbdd-4963-4bbe-ac9c-01e3c70da1ea-attachment.txt", "type": "text/plain"}], "start": 1653572140643, "stop": 1653572171476, "uuid": "e967af35-f178-4b49-89b6-77b0f6f08c5c", "historyId": "aeb6c84858cb897d59574054d46d09e3", "testCaseId": "1512c76b4b2d60f059541c0f73c45353", "fullName": "TestSuite.LoginTest#test_loginFailMethod", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('beforeClass', 'beforeMethod')"}, {"name": "tag", "value": "run(order=1)"}, {"name": "suite", "value": "TestSuite"}, {"name": "subSuite", "value": "LoginTest"}, {"name": "host", "value": "BTZVBK3"}, {"name": "thread", "value": "15080-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestSuite"}]}